lib/parser/builders/default.rb

```diff
@@ -2295,10 +2295,14 @@ def collapse_string_parts?(parts)
           [:str, :dstr].include?(parts.first.type)
     end
 
+    # @param [::Parser::AST::Node] token
```

Comment:
Sorry for a late review but I think this signature is wrong. This method takes a token, not a node. Same for the method below.